src/evict/evict_lru.c

```diff
@@ -277,6 +277,9 @@ __wt_evict_thread_run(WT_SESSION_IMPL *session, WT_THREAD *thread)
     conn = S2C(session);
     cache = conn->cache;
 
+    /* Mark the session as an eviction thread session. */
+    F_SET(session, WT_SESSION_EVICTION);
```

Comment:
@quchenhao The change is merged, so I missed the chance, but I think the name of this flag could be better.

I see this and think that any session that is currently doing an eviction, or any session that is prepared to do an eviction might have the flag set. Not necessarily that this session is associated with a dedicated eviction worker. A better name would make the flag better.

I don't feel strongly enough to open a follow on ticket, but wanted to mention it.

Reply:
Thanks, got it.